cfme/tests/networks/test_network_providers.py

```diff
@@ -51,7 +51,9 @@ def test_network_provider_add_with_bad_credentials(provider):
     default_credentials = provider.default_endpoint.credentials
 
     # default settings
-    flash = 'Login failed due to a bad username or password.'
+    flash = 'Login failed due to a bad username, password or unsupported API version.'
+    if provider.appliance < 5.10:
+        flash = 'Login failed due to a bad username or password.'
```

Comment:
Close! Here you are getting the IPappliance object. 
```In[31]: provider.appliance
Out[31]: IPAppliance.from_json('{"container": null, "ui_port": 443, "ssh_port": 22, "openshift_creds": {}, "ui_protocol": "https", "hostname": "10.8.198.243", "db_port": 5432, "project": null, "is_dev": false, "version": "5.10.0.29", "db_host": null, "browser_steal": false}')
```
You will want to go one step further and call the version property:
```In[30]: provider.appliance.version
Out[30]: Version('5.10.0.29')
```

Reply:
oh, yes, of course ... Should be good now :)

Reply:
and could you put both statements in one if/else block to make it more neat

Reply:
done :)